packages/typespec-test/test/contentsafety_modular/generated/typespec-ts/src/api/contentSafetyContext.ts

```diff
@@ -1,9 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { TokenCredential, KeyCredential } from "@azure/core-auth";
-import { ClientOptions, Client, getClient } from "@azure-rest/core-client";
+import { KeyCredential, TokenCredential } from "@azure/core-auth";
```

Comment:
in this case,  I found if I run this case alone, it finishes successfully, however if I run it in the smoke-test batch list, it fails at KeyCredential and TokenCredential is not imported, @joheredi, do you have any clues about this? I feel like maybe this is because Binder is a singleton instance which may not be able to work in parallel mode?